src/app/home/invitation/invitation.component.scss

```diff
@@ -1,33 +1,4 @@
-@import '../../../styles.colors';
-@import '../../../styles.variables';
-
-.invitation-container {
-  object-fit: contain;
-  padding-top: 100px;
-  padding-bottom: 40px;
-}
-
-.title {
-  color: $primary-color;
-  font-size: 34px;
-  font-weight: bold;
-  font-family: 'Open Sans', serif;
-}
-
-.description {
-  font-size: 18px;
-}
-
-.section {
-  margin-top: 40px;
-  margin-bottom: 40px;
-}
-
 .text-section {
-  font-size: 20px;
+  font-size: 1.4285rem;
   width: 80%;
```

Comment:
I don't really sure how browser render text but add `width: 80%` it's ok? In not sure if is apply in a container or in the same text 🙃

Reply:
It should be fine to apply the width directly to the text, but I agree with placing this text and the image besides it inside containers and apply the styles there, I'll work on that

Reply:
oh ok, in mobile is a recalculate and scaling is not performant 🙈, another type of problems 🙈